skore/src/skore/sklearn/_base.py

```diff
@@ -394,13 +395,16 @@ def _get_cached_response_values(
         assert isinstance(cached_predictions, np.ndarray)
         return cached_predictions
 
-    predictions, _ = _get_response_values(
-        estimator,
-        X=X,
-        response_method=prediction_method,
-        pos_label=pos_label,
-        return_response_method_used=False,
-    )
+    with _measure_time() as predict_time:
+        predictions, _ = _get_response_values(
+            estimator,
+            X=X,
+            response_method=prediction_method,
+            pos_label=pos_label,
+            return_response_method_used=False,
+        )
+
     cache[cache_key] = predictions
+    cache[cache_key + ("predict_time",)] = predict_time()
```

Comment:
I'm thinking that we could simplify the cache key here:

```python
(estimator_hash, data_source, data_source_hash, "predict_time")
```

would be enough. We don't really care that we computed `predict` or `predict_proba` or `decision_function`. Usually the difference between them will be negligeable. This would have the benefit to simplify the search later